src/test/java/io/confluent/examples/streams/kafka/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -220,22 +216,24 @@ public void createTopic(final String topic, final int partitions, final int repl
    */
   public void createTopic(final String topic,
                           final int partitions,
-                          final int replication,
-                          final Properties topicConfig) {
+                          final short replication,
+                          final Map<String, String> topicConfig) {
     broker.createTopic(topic, partitions, replication, topicConfig);
   }
 
   /**
    * Deletes multiple topics and blocks until all topics got deleted.
    *
    * @param timeoutMs the max time to wait for the topics to be deleted (does not block if {@code <= 0})
-   * @param topics the name of the topics
+   * @param topics    the name of the topics
    */
   public void deleteTopicsAndWait(final long timeoutMs, final String... topics) throws InterruptedException {
     for (final String topic : topics) {
       try {
         broker.deleteTopic(topic);
-      } catch (final UnknownTopicOrPartitionException e) { }
+      } catch (final UnknownTopicOrPartitionException e) {
+        // indicates success
```

Comment:
nit: improve variable name `e` instead of adding a comment?

Reply:
I'll improve the variable name, but the comment is there to avoid a compiler warning regarding empty catch blocks.